src/Sylius/Bundle/GridBundle/Doctrine/DBAL/Driver.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\GridBundle\Doctrine\DBAL;
+
+use Doctrine\DBAL\Connection;
+use Sylius\Component\Grid\Data\DriverInterface;
+use Sylius\Component\Grid\Parameters;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class Driver implements DriverInterface
+{
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    /**
+     * @param Connection $connection
+     */
+    public function __construct(Connection $connection)
+    {
+        $this->connection = $connection;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDataSource(array $configuration, Parameters $parameters)
+    {
+        if (!array_key_exists('table', $configuration)) {
```

Comment:
`if (!isset($configuration['table'])) { /* ... */ }` should be faster (and more readable [at least for me]).

If we merge the PR introducing the assertion library (#4385), then we can use `Assert::keyExists($configuration, 'table', 'our message')`.